### ISSUE TITLE
Icon library

### ISSUE THREAD

**Issue by christiemolloy** [uxd-mk-prototype/uxd-mk-data-plane-ui#11]
What are the icons used here? They dont look like they're from the Font Awesome library.

https://marvelapp.com/prototype/26598gag/screen/73803099

**Comment by lboehling** [comment]
@christiemolloy those are svg illustrations provided by IBM. They have been approved to be used in the upstream PF version of Strimzi. 

Attaching them here: [pictograms.zip](https://github.com/uxd-mk-prototype/uxd-mk-data-plane-ui/files/5441711/pictograms.zip)

**Comment by christiemolloy** [comment]
Thanks @lboehling, also looking at the icon name and the actual icon, it looks like Documentation and Create should maybe be swapped around?

<img width="560" alt="Screen Shot 2020-10-27 at 10 05 25 AM" src="https://user-images.githubusercontent.com/20118816/97312599-0b341880-183c-11eb-80bc-e155ed8c3d04.png">

**Comment by lboehling** [comment]
Yep, good catch! those should be switched. thanks @christiemolloy
